### ISSUE TITLE
Multiple chat windows tab

### ISSUE THREAD

**Issue by mmmikeal** [dharness/react-chat-window#30]
Would be nice for Launcher to take an array of multiple chats and make tabs for each chat with each unique user

**Comment by dharness** [comment]
Yeah totally agree, that's what the genuine Intercom has and it should be on the roadmap. Realistically I won't get to that feature for several months.

**Comment by dharness** [comment]
I'm removing this from the next release. I haven't heard anyone else require it, and it's a fair amount of work.

**Comment by lineldcosta** [comment]
It would be nice to have this feature.. Even i am looking for it... tried in different ways but couldn't give that good feel...

**Comment by joemillervi** [comment]
++ This feature would be sweet

**Comment by flamebeard** [comment]
Also very interested in this feature

**Comment by rudden** [comment]
Same!

**Comment by ko06** [comment]
Also interested!

**Comment by ko06** [comment]
@lineldcosta can you share your  code/ approach

**Comment by kimgysen** [comment]
Cannot use the lib because of this missing feature, otherwise wouldn't have doubted. I feel it's better not to wait for user feedback, many users will just move on without leaving feedback. This is the only lib available that does the private chat window thingy well. I'll be forced to go with a terrible accordion UI approach, but at least I can chat with multiple users.
